README.md

```diff
@@ -101,6 +101,21 @@ Auto-deploying manifests
 Any file found in `/var/lib/rancher/k3s/server/manifests` will automatically be deployed to
 Kubernetes in a manner similar to `kubectl apply`.
 
+It is also possible to deploy Helm charts. k3s supports a CRD controller for installing charts. A YAML file specification can look as following (example taken from `/var/lib/rancher/k3s/server/manifests/traefik.yaml`):
+
+```
+apiVersion: k3s.cattle.io/v1
+kind: HelmChart
+metadata:
+  name: traefik
+  namespace: kube-system
+spec:
+  chart: stable/traefik
+  set:
+    rbac.enabled: "true"
+    ssl.enabled: "true"
+```
+
```

Comment:
Do you want to put a reference to https://github.com/rancher/k3s/blob/master/types/apis/k3s.cattle.io/v1/types.go here. The type HelmChartSpec has the full reference of all fields that can be set in the Custom Resource.